presto-test-jdbc-compatibility/src/test/java/io/prestosql/TestJdbcCompatibility.java

```diff
@@ -49,9 +51,17 @@
 import static io.prestosql.JdbcDriverCapabilities.testedVersion;
 import static java.lang.Integer.min;
 import static java.lang.String.format;
+import static java.sql.Types.ARRAY;
 import static java.sql.Types.JAVA_OBJECT;
+import static java.sql.Types.TIMESTAMP;
 import static org.assertj.core.api.Assertions.assertThat;
 
+/**
+ * Main purpose of this class is to test cases when current server implementation breaks older JDBC clients
+ * to ensure that current implementation is backward-compatible.
+ *
+ * TestJdbcResultSetCompatibility in that regard is responsible for testing forward compatibility.
```

Comment:
thanks

make TestJdbcResultSetCompatibility a `{@link`

Add a note that `TestJdbcResultSetCompatibility` uses the old test code from old presto version against new server and this class is a new test code that's used against old JDBC and the new server.

Also, maybe we can rename the classes to indicate that? Thoughts?